Source/FieldSolver/ElectrostaticSolvers/RelativisticExplicitES.cpp

```diff
@@ -133,12 +133,12 @@ void RelativisticExplicitES::AddSpaceChargeField (
                 pc.self_fields_verbosity );
 
     // Compute the corresponding electric and magnetic field, from the potential phi
-    computeE( Efield_fp, amrex::GetVecOfPtrs(phi), beta );
-    computeB( Bfield_fp, amrex::GetVecOfPtrs(phi), beta );
+    computeE( E_fp, amrex::GetVecOfPtrs(phi), beta );
+    computeB( B_fp, amrex::GetVecOfPtrs(phi), beta );
 
 }
 
-void RelativisticExplicitES::AddBoundaryField (ablastr::fields::MultiLevelVectorField& Efield_fp)
+void RelativisticExplicitES::AddBoundaryField (ablastr::fields::MultiLevelVectorField& E_fp)
```

Comment:
Clang-Tidy: `.H` file that defines this function should also rename the argument name to `E_fp`.

Reply:
Pushed